<launch>
  <arg name="cam_2" default="camera_2" />
  <node pkg="gige_cam_driver" type="camera_node.py" name="$(arg cam_2)" output="screen">
      <param name="device_id" value="1"/>
      <param name="camera_manager" value="$(arg cam_2)"/>
      <param name="calibration_file" value="$(find gige_cam_driver)/config/camera_info_cam2.yaml"/>
   </node>
  <node pkg="camera_calibration" name="camera_2_calibration" type="cameracalibrator.py" output="screen">
    <param name="size" value="6x5"/>
    <param name="square" value="0.025"/>
    <param name="k-coefficients" value="2"/>
    <param name="fix-principal-point" value="i"/>
    <param name="fix-aspect-ratio" value="true"/>
    <remap from="image" to="/camera_2/image_raw"/>
    <remap from="camera" to="/camera_2"/>
  </node>
</launch>